src/transports/transports.ts

```diff
@@ -1,16 +1,17 @@
 // System packages
 import fs from "fs/promises";
 import path from "path";
-import { LogEntry } from "./log-entry";
-import { db } from "./firebase/firebase";
-import { doc, setDoc } from "firebase/firestore";
-
-import dotenv from "dotenv";
-dotenv.config();
 
 // Local dependencies
 import { LogEntry } from "../log-entry";
 
+// Third party packages
```

Comment:
These imports don't seem used. I also don't see the utility of this file, okay to delete.

Reply:
They are used. The file is where we are going to implement transport for json array, and where consoleTransport and fileTransport